src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc

```diff
@@ -90,10 +92,14 @@ void SDLActivateAppRequest::Run() {
     }
     subscribe_on_event(BasicCommunication_OnAppRegistered);
   } else {
-    const uint32_t application_id = app_id();
-    application_manager::ApplicationManagerImpl::instance()
-	->GetPolicyHandler().OnActivateApp(application_id,
-					   correlation_id());
+    if (devices_apps.first) {
+      MessageHelper::SendLaunchApp(
+          devices_apps.first->app_id(), app->SchemaUrl(), app->PackageName());
```

Comment:
@AGritsevich are you sure we have to send `LAUNCH_APP` to already registered application?